server/eventlog.go

```diff
@@ -88,13 +88,37 @@ func parsePCClientEventLog(rawEventLog []byte, pcrs *tpmpb.PCRs, loader Bootload
 	}, createGroupedError("failed to fully parse MachineState:", errors)
 }
 
-func parseCanonicalEventLog(rawCanonicalEventLog []byte, pcrs *tpmpb.PCRs) (*pb.MachineState, error) {
+func parseCanonicalEventLogPCR(rawCanonicalEventLog []byte, pcrs *tpmpb.PCRs) (*pb.MachineState, error) {
+	tpm2Alg := tpm2.Algorithm(pcrs.GetHash())
+	cryptoHash, err := tpm2Alg.Hash()
+	if err != nil {
+		return nil, err
+	}
+	return getCosStateFromCEL(rawCanonicalEventLog, cryptoHash, pcrs.GetPcrs())
+}
+
+func parseCanonicalEventLogRTMR(rawCanonicalEventLog []byte, rtmrs [][]byte) (*pb.MachineState, error) {
```

Comment:
Two things:
1. could you export this function because GAV will likely consume it.
2. From GAV perspective, what's the source of `rtmrs`? I know `rawCanonicalEventLog` comes from this proto field(https://source.corp.google.com/piper///depot/google3/google/cloud/confidentialcomputing/v1main/service.proto;rcl=650697927;l=231), but where can I locate `rtmrs` which is a digest array?

Reply:
RTMRs should be read from the TD quote. https://github.com/google/go-tdx-guest/blob/main/rtmr/ccel.go#L58

Reply:
+1 to Yawang. Could you export this function?

Reply:
You should just use the register.RTMRBank type since that's what go-eventlog and go-tdx-guest use.

Reply:
> * could you export this function because GAV will likely consume it.

I thought this function is only going to be consumed by the server.VerifyAttestation? But sure I can export it.

Reply:
GAV will consume this function to parse and verify CosState from CEL logs for TDX RTMR. Also looks like this function is not called by `server.VerifyAttestation` from your PR changes.. so I think export is needed here.